addons/account_edi/controller/main.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import http
+from odoo.http import request, content_disposition
+
+
+class EdiDocumentDownloadController(http.Controller):
+    @http.route('/web/edi_document/download', type='http', auth='user')
+    def download(self, ids=None):
+        ids = [int(id) for id in ids.split(',')]
+        res = request.env['account.move'].sudo().browse(ids)._create_zipped()
+        content = res.getvalue()
+        headers = [
+            ('Content-Type', 'zip'),
+            ('X-Content-Type-Options', 'nosniff'),
+            ('Content-Length', len(content)),
+            ('Content-Disposition', content_disposition('edi_documents.zip'))
```

Comment:
```suggestion
            ('Content-Disposition', content_disposition(f'{_("edi_documents")}.zip'))
```
We may want to let the zip name be translatted